pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -183,6 +182,7 @@ const CryptoPolicyDialog = ({
                    </>
                }
         >
+            {error && <ModalError dialogError={typeof error == 'string' ? error : error.message} />}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17221-20221202-001251-c9401003-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)